dotcom-rendering/src/web/components/KeyEventsCarousel.importable.tsx

```diff
@@ -139,6 +140,7 @@ export const KeyEventsCarousel = ({
 								}
 								isSummary={keyEvent.attributes.summary}
 								title={keyEvent.title}
+								cardPosition={`${i} / ${carouselLength}`}
```

Comment:
Ah, this was supposed to read as `{cardindex} of {total length}` (eg. 1 of 10) rather than a division symbol. Thanks for the comment, I'll make this more explicit!